<launch>

  <node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen" clear_params="true" respawn ="true">
    <param name="~port" value="/dev/ttyACM0" />
    <param name="~baud" value="57600" />
  
  </node>

</launch>
